### DISCUSSION TITLE
유니티에서도 빌드잇 UI 적용

### DISCUSSION THREAD

**Discussion by DongKey123** [naverz/zepeto-studio-kor#324] [5. Q&A]
![image](https://user-images.githubusercontent.com/21350576/152573757-8fec3711-6b88-433d-a14a-ecd4350d65d3.png)
유니티에서도 이런 UI를 적용할 수 있을까요?

**Comment by x4cf** [comment]
당장에는 불가능합니다

**Comment by hyeonjin-jo** [comment]
안녕하세요.

현재는 해당 기능들을 제공하고 있지 않습니다.

이후 해당 기능 추가에 대해서 논의해 보겠습니다

**Reply by DongKey123** [reply]
롯데월드 맵의 경우에는 유니티로 제작되었다고 들었는데 UI는 위와 같이 적용이 되었어서 혹시 직접구현으로도 가능 할까요?

**Reply by x4cf** [reply]
불가능합니다

**Reply by DongKey123** [reply]
넵 감사합니다.

**Reply by DongKey123** [reply]
혹시 롯데월드 맵의 경우 빌드잇으로 제작되었는 지 제페토로 제작되었는 지 알 수 있을까요?
